ledger/db/src/ledger_db.rs

```diff
@@ -32,12 +33,15 @@ use std::{
 };
 
 pub const MAX_LMDB_FILE_SIZE: usize = 1 << 40; // 1 TB
-pub const MAX_LMDB_DATABASES: u32 = 27; // maximum number of databases in the lmdb file
+
+/// maximum number of [Database]s in the lmdb file
+pub const MAX_LMDB_DATABASES: u32 = 19;
```

Comment:
did we really go down from 27 to 19 here? or this number was just wrong before?

Reply:
The number was wrong before. I tried iteratively lowering it, 18 failed but 19 passed.